config/locales/en/course/statistics.yml

```diff
@@ -1,4 +1,7 @@
 en:
   course:
     statistics:
+      name: 'Name'
+      status: 'Status'
```

Comment:
Nit: this is the course_user `role` not `status`

Reply:
done

Reply:
Sorry my mistake, this should be the Student Type (following `/statistics/students` page)

Reply:
done